src/pages/FilteredEvents.jsx

```diff
@@ -0,0 +1,57 @@
+import EventCard from '../components/EventCardTest/EventCard';
+import { useEffect, useState } from 'react';
+import { getEvents } from '../utils/eventsUtils';
+import { SimpleGrid } from '@chakra-ui/react';
+
+const FilteredEvents = () => {
+  const [events, setEvents] = useState([]);
+  // Array of all events from DB
+  const [isAdmin, setAdminStatus] = useState([]);
+  // Bool, if user is admin or not
+  const [pastEvents, setPastEvents] = useState([]);
+  const [futureEvents, setFutureEvents] = useState([]);
+  const [currentEvents, setCurrentEvents] = useState([]);
+
+  const loadData = async () => {
+    await getEvents().then(data => setEvents(data));
+    const date = new Date(Date.UTC(2012, 11, 20, 3, 0, 0, 200));
+    let options = {
+        timeZone: "PST"
+    };
+    const dateTime = new Intl.DateTimeFormat(options).format(date);
+
+    for (let eventIndex = 0; eventIndex < events.length; eventIndex++) {
+        const event = events[eventIndex];
+
+        //2024-02-18T08:00:00.000Z
+        if (event.date < dateTime) {
+            // Past event
+            setPastEvents(pastEvents + event);
+        } else if (event.date > dateTime) {
+            // Future Event
+            setFutureEvents(futureEvents + event);
+        } else {
+            // Current
+            setCurrentEvents(currentEvents + event);
+        }
+    }
+
+    setAdminStatus(true);
+  };
+
+  useEffect(() => {
+    loadData();
+  }, []);
```

Comment:
⚠️ **[eslint]** <[react-hooks/exhaustive-deps](https://github.com/facebook/react/issues/14920)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>React Hook useEffect has a missing dependency: 'loadData'. Either include it or remove the dependency array.